packages/fxa-content-server/package.json

```diff
@@ -38,6 +38,7 @@
         "@babel/preset-env": "7.6.2",
         "@babel/preset-react": "^7.0.0",
         "@babel/preset-typescript": "^7.3.3",
+        "@sentry/browser": "5.6.3",
```

Comment:
Do [`raven` and `raven-js`](https://github.com/mozilla/fxa/pull/3388/files#diff-57ab2cfbf986be5762b650f526ffb546R112) need to be removed?

Reply:
raven-js does need to be removed, I also need to remove it in the webpack config. I didn't touch the places that are using raven directly as this issue was only to get sourcemaps working on the client code. I'd like to completely remove it in the future, but it will be a tad more refactoring, so I'd like to get this in so we can do our sentry triage this wednesday, and follow up with the rest of raven removal in a follow up bug.

I also have a couple references to raven-js in a couple of tests that should be removed before this is merged. Thanks for catching this :)